internal/gitlab/gitlab.go

```diff
@@ -81,6 +81,18 @@ var (
 	localProjects map[string]*gitlab.Project = make(map[string]*gitlab.Project)
 )
 
+// GetProject looks up a Gitlab project by ID.
+func GetProject(projectID int) (*gitlab.Project, error) {
```

Comment:
Maybe we should just update `FindProject` to accept an `interface{}` and `switch` for the type. `go-gitlab` will accept the interface form as well

Reply:
After putting 15min into this. I request some help. The problem is resolving the whole `localProject` map.

How about I add a TODO? Maybe an issue as well? Or we could pair on it.

Reply:
After chatting with you more (in person) this is fine as is.